src/viewer_settings.ts

```diff
@@ -38,22 +37,6 @@ export const ViewerSettings = {
         z: 0
       }
     },
-    sunlight: {
-      skyColor: { r: 0x44, g: 0x33, b: 0x33 },
-      groundColor: { r: 0x11, g: 0x11, b: 0x22 },
-      intensity: 1
-    },
-    light1: {
-      // TODO: the positions of the lights are all wrong.
-      position: { x: 1, y: 1, z: 1 },
-      color: { r: 0xff, g: 0xff, b: 0xff },
-      intensity: 1.35
-    },
-    light2: {
-      position: { x: 0.5, y: 1, z: -1 },
-      color: { r: 0xff, g: 0xaa, b: 0x00 },
-      intensity: 1
-    },
```

Comment:
It seems like we're removing the default values for `sunlight`, `light1`, and `light2` here, and instead hard-coding new default values in `viewer.ts`. To keep things flexible from a configuration standpoint, would it make sense to restore these default values in this object and read them in `viewer.ts`?

Reply:
Yes, I agree.